includes/migrations/class-erase-deprecated-flags-and-options.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Class Erase_Deprecated_Flags_And_Options
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Migrations;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class Erase_Deprecated_Flags_And_Options
+ *
+ * Clearing up all flags and options that are no longer in use.
+ *
+ * @since 8.1.0
+ */
+class Erase_Deprecated_Flags_And_Options {
+	/**
+	 * Checks whether it's worth doing the migration.
+	 */
+	public function maybe_migrate() {
+		$previous_version = get_option( 'woocommerce_woocommerce_payments_version' );
+		// feel free to modify the version here to the next one, if you add a new flag to be deleted in the `migrate` method.
+		if ( version_compare( '8.1.0', $previous_version, '>' ) ) {
+			$this->migrate();
+		}
+	}
+
+	/**
+	 * Does the actual migration.
+	 */
+	private function migrate() {
+		// please update as necessary, when feature flags are removed.
+		// you don't need to check for previous values or if the migration already ran in previous versions.
+		// if the migration did already run, the operations below will just be noop.
+		delete_option( '_wcpay_feature_progressive_onboarding' );
+		delete_option( '_wcpay_feature_client_secret_encryption' );
+		delete_option( '_wcpay_feature_allow_subscription_migrations' );
+		delete_option( '_wcpay_feature_custom_deposit_schedules' );
+		delete_option( '_wcpay_feature_account_overview_task_list' );
+		delete_option( '_wcpay_feature_account_overview' );
+		delete_option( '_wcpay_feature_sepa' );
+		delete_option( '_wcpay_feature_sofort' );
+		delete_option( '_wcpay_feature_giropay' );
+		delete_option( '_wcpay_feature_grouped_settings' );
+		delete_option( '_wcpay_feature_customer_multi_currency' );
```

Comment:
It looks like this flag is still being used.

https://github.com/Automattic/woocommerce-payments/blob/06ede9f56a7e1a0142b5c632a345da36728f8483/includes/class-wc-payments-features.php#L73-L75

https://github.com/Automattic/woocommerce-payments/blob/06ede9f56a7e1a0142b5c632a345da36728f8483/includes/admin/class-wc-rest-payments-settings-controller.php#L750

Reply:
🤦 I thought I double-checked everything, thank you for catching it!
```suggestion
```